tools/strip_nibabel

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/python
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+#
+#   See COPYING file distributed along with the PyMVPA package for the
+#   copyright and license terms.
+#
+### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+"""Convert NiBabel objects in PyMVPA datasets attributes into a portable form
```

Comment:
btw by placing it under mvpa2/ codebase we at least would have better opportunity to provide a unittest for this beastie. Especially if that PR for testing compatibility would get merged -- i.e. if we get our test files for this conversion to be tested